packages/formatter-excel/package.json

```
@@ -8,7 +8,7 @@
   },
   "description": "webhint formatter that outputs the result in an Excel file (xlsx)",
   "dependencies": {
-    "exceljs": "^1.7.0",
+    "exceljs": "^1.9.0",
```

Hi. I saw your problem in the [exceljs issue tracker](https://github.com/exceljs/exceljs/issues/773). That should fix your issue, if you can live with using 1.8.0
```suggestion
    "exceljs": "1.8.0",
```

---

Thanks!
We don't add any images so 1.7.0 is fine for us, it's just that we have dependabot updating everything and we want to know the reason when it fails so we can fix our code or wait for a new version if needed 😊

---

Changed the version to be 1.8.0, the error is still on.